src/coreclr/pal/src/exception/machmessage.cpp

```diff
@@ -852,6 +852,11 @@ thread_act_t MachMessage::GetThreadFromState(thread_state_flavor_t eFlavor, thre
                 CHECK_MACH("mach_port_mod_refs()", machret);
```

Comment:
I can probably skip deallocation for `pThreads[i]` and remove this manual increment, if needed.

Reply:
I actually prefer the current form.